### DISCUSSION TITLE
how to create a character if I don't have a license?

### DISCUSSION THREAD

**Discussion by jooperka** [ORelio/Minecraft-Console-Client#2647] [Q&A]
How to create a character if I don't have a license. It gives me an error
Received error code from server: 410
Minecraft login failed: Network error.

Not connected to any server. Use '/help' for help.
Press Enter to exit the Minecraft console client.

I'm using a slightly earlier version, but the new one has this error

**Comment by breadbyte** [comment]
Simply press enter when it asks you for a password.
